modelscope_agent/agent_env_util.py

```diff
@@ -0,0 +1,297 @@
+import os
+import time
+from pathlib import Path
+from typing import Callable, List, Optional, Union
+
+import ray
+from modelscope_agent.constants import DEFAULT_AGENT_ROOT, DEFAULT_SEND_TO
+from modelscope_agent.environment import Environment
+from modelscope_agent.memory import MemoryWithRetrievalKnowledge
+from modelscope_agent.schemas import Message
+from modelscope_agent.utils.logger import agent_logger as logger
+from ray._raylet import ObjectRefGenerator
+from ray.util.client.common import ClientActorHandle, ClientObjectRef
```

Comment:
this module should not be always depends on ray, make it optional

Reply:
removed